hardhat.config.ts

```diff
@@ -86,14 +88,40 @@ const config: HardhatUserConfig = {
         ? [process.env.BASE_SEPOLIA_DEPLOYER_PRIVATE_KEY]
         : [],
     },
+    base: {
+      chainId: 8453,
+      url: process.env.BASE_PROVIDER || "",
+      accounts: process.env.BASE_DEPLOYER_PRIVATE_KEY
+        ? [process.env.BASE_DEPLOYER_PRIVATE_KEY]
+        : [],
+    },
   },
   etherscan: {
     apiKey: {
       mainnet: process.env.ETHERSCAN_API_KEY || "",
       sepolia: process.env.ETHERSCAN_API_KEY || "",
       polygon: process.env.POLYGONSCAN_API_KEY || "",
       baseSepolia: process.env.BASESCAN_API_KEY || "",
+      base: process.env.BASESCAN_API_KEY || "",
     },
+    customChains: [
+      {
+        network: "baseSepolia",
+        chainId: 84532,
+        urls: {
+          apiURL: "https://api-sepolia.basescan.org/api",
+          browserURL: "https://sepolia.basescan.org",
+        },
+      },
+      {
+        network: "base",
+        chainId: 8453,
+        urls: {
+          apiURL: "https://api.basescan.org/api",
+          browserURL: "https://basescan.org",
+        },
+      },
+    ],
```

Comment:
Whatever version of `hardhat-verify` we're using doesn't have support for these Base etherscan instances, but luckily we can just add any custom ones we need to. Seemed easier than trying to update dependencies.

Reply:
After doing this, I finished verifying contracts on Base Sepolia (some were already verified...) and verified all contracts on Base that I just deployed.